core/src/main/java/org/apache/carbondata/core/index/TableIndex.java

```diff
@@ -187,8 +191,18 @@ public CarbonTable getTable() {
     return blocklets;
   }
 
+  private Set<Path> getPartitionsToPrune(List<PartitionSpec> partitionSpecs) {
```

Comment:
i think its better to rename the method name as `getPartionLocations`

Reply:
done